frontend/src/metabase/visualizations/components/ObjectDetail/utils.unit.spec.ts

```diff
@@ -32,57 +35,63 @@ const card = {
 } as Card;
 
 describe("ObjectDetail utils", () => {
-  const productIdCol = {
+  const productIdCol = createMockColumn({
     name: "product_id",
     display_name: "Product ID",
     base_type: "int",
     effective_type: "int",
     semantic_type: "type/PK",
     table_id: PRODUCTS.id,
-  };
-  const idCol = {
+  });
+
+  const idCol = createMockColumn({
     name: "id",
     display_name: "ID",
     base_type: "int",
     effective_type: "int",
     semantic_type: "type/PK",
     table_id: ORDERS.id,
-  };
-  const remappedIdCol = {
+  });
+
+  const remappedIdCol = createMockColumn({
```

Comment:
Seems like it is not a column but a `Field` since you assign `hasRemappedValue` below, wdyt about using the Field class here?